test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/ModelExpressionTagHelper_Runtime.ir.txt

```diff
@@ -27,14 +28,15 @@ Document -
                 CSharpCode - 
                     IntermediateToken -  - CSharp - BeginContext(66, 25, false);
                 TagHelper - (66:4,0 [25] ModelExpressionTagHelper.cshtml) - input-test - TagMode.SelfClosing
-                    TagHelperBody - 
-                    CreateTagHelper -  - InputTestTagHelper
-                    SetTagHelperProperty - (83:4,17 [4] ModelExpressionTagHelper.cshtml) - for - For - HtmlAttributeValueStyle.DoubleQuotes
+                    DefaultTagHelperBody - 
+                    DefaultTagHelperCreate -  - InputTestTagHelper
+                    DefaultTagHelperProperty - (83:4,17 [4] ModelExpressionTagHelper.cshtml) - for - Microsoft.AspNetCore.Mvc.ViewFeatures.ModelExpression InputTestTagHelper.For - HtmlAttributeValueStyle.DoubleQuotes
```

Comment:
I see some inconsistencies where some places we use `DefaultTagHelperProperty` and other we use `DefaultTagHelperPropertyValue`. We should make them all be the same.